tests/draft2019-09/optional/ecmascript-regex.json

```diff
@@ -1,11 +1,29 @@
 [
     {
         "description": "ECMA 262 regex non-compliance",
-        "schema": { "format": "regex" },
+        "schema": {
+            "type": "string",
+            "pattern": "^\\S\\Z"
+        },
         "tests": [
             {
-                "description": "ECMA 262 has no support for \\Z anchor from .NET",
-                "data": "^\\S(|(.|\\n)*\\S)\\Z",
+                "description": "\\Z should match literal Z",
+                "data": "xZ",
+                "valid": true
+            },
+            {
+                "description": "\\Z should not match end",
+                "data": "x",
+                "valid": false
+            },
+            {
+                "description": "\\z should match literal z",
+                "data": "xz",
+                "valid": true
+            },
+            {
+                "description": "\\z should not match end",
+                "data": "x",
```

Comment:
these two are for lowercase `\z` but the pattern that applies to them only contains uppercase `\Z` - should be split into  a separate test?

Reply:
You’re right. I’m not sure what I was thinking there.

Reply:
I'd personally love to see all these special-case Javascript cases removed. I just didn't think "^\\S\\Z" was useful if the test was just parsing javascript and not testing execution.